lib/challenge.js

```diff
@@ -229,15 +285,34 @@ let nums = [
 ];
 
 function prompt11() {
-  let median;
-  // YOUR CODE HERE
-
-  console.log(median);
+  // let median;
+  // // YOUR CODE HERE
+  // nums.sort(function (a, b) { return a - b });
+  // let i = Math.ceil(nums.length / 2);
+  // median = nums[i];
+  // console.log(median);
 }
```

Comment:
Nice start!  What about if the length of the array is even? (resubmission not required)